data/json/items/book/spiritual.json

```
@@ -1305,7 +1305,7 @@
   {
     "id": "holybook_golden",
     "type": "BOOK",
-    "name": { "str": "The Orphic Gold Tablets", "str_pl": "copies of the The Orphic Gold Tablets" },
+    "name": { "str": "The Orphic Gold Tablets", "str_pl": "copies of the Orphic Gold Tablets" },
```

```suggestion
    "name": { "str": "The Orphic Gold Tablets", "str_pl": "copies of The Orphic Gold Tablets" },
```
`The` is part of the book title.